client/client-multi/client-java-core/src/main/java/org/kaaproject/kaa/client/logging/future/BucketFuture.java

```diff
@@ -0,0 +1,61 @@
+package org.kaaproject.kaa.client.logging.future;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.*;
+
+public class BucketFuture<T> implements Future<T> {
```

Comment:
It may cause issues on Android. http://stackoverflow.com/questions/23318109/is-it-possible-to-use-java-8-for-android-development